spikingjelly/activation_based/ann2snn/operators.py

```
@@ -1200,6 +1745,8 @@ def __init__(
         神经形态硬件的 fully spike-driven MultiheadAttention。``bias=True``
         时 projection bias 由 ``TDLinear`` 在累积输入上处理，避免普通
         ``nn.Linear`` 直接作用在差分序列时产生重复累计 bias。
+        父模块的 ``step_mode`` 会同步到内部 q/k/v/out projection，确保 projection
```

Ruff (0.15.18)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)